src/dtos/user-dto.ts

```diff
@@ -0,0 +1,19 @@
+import { User } from '../entity/user';
+
+export class UserDto {
+    id: string;
+    oidcId: string;
+    provider: string;
+    name: string;
```

Comment:
In the general case this can change at every login. We need to decide if we care about a history of changes or if there is any user facing thing we need to do when it happens.